package/MDAnalysis/transformations/translate.py

```diff
@@ -21,50 +21,120 @@
 #
 
 """\
-Translations --- :mod:`MDAnalysis.transformations.translate`
-============================================================
+Trajectory translation --- :mod:`MDAnalysis.transformations.translate`
+======================================================================
 
 Translate the coordinates of a given trajectory by a given vector.
-
-.. autofunction:: translate   
-    
+The vector can either be user defined, using the function :func:`translate`
+or defined by centering an AtomGroup in the unit cell using the function
+:func:`center_in_box`
     
 """
-from __future__ import absolute_import
+from __future__ import absolute_import, division
 
 import numpy as np
+from functools import partial
+
+from ..lib.mdamath import triclinic_vectors
+from ..core.groups import AtomGroup
```

Comment:
I'm not sure you need this AtomGroup import here, maybe it's old.

In general these transformations should use duck typing, ie not explicitly checking the type of things coming in, but simply expecting them to do all the things an AtomGroup does.

Reply:
Indeed, the import is not used anymore.